app/experimenter/nimbus-ui/src/lib/dateUtils.ts

```diff
@@ -12,9 +12,15 @@ export function humanDate(
 ): string {
   const today = new Date();
   const parsedDate = new Date(date);
+  // Dates currently arrive both with and without time and
+  // timezone, so reset time to start of day and force GMT
+  parsedDate.setUTCMinutes(0);
+  parsedDate.setUTCHours(0);
+  parsedDate.setUTCSeconds(0);
```

Comment:
Super-picky, but maybe we also want to add `parseDate.setUTCMilliseconds(0)` here too? Technically we could still end up with a start date that's after the end date if they're not absolutely equal. But I'm hard-pressed to come up with a real problem millisecond difference would cause.